### DISCUSSION TITLE
Request for more specific instructions on method of usage

### DISCUSSION THREAD

**Discussion by Tomas2133** [KJHJason/Cultured-Downloader#191] [Q&A]
Am new to programming as a whole. Not sure what to do with the cultured downloader after starting up the program and specifying the directory. Need further instructions on usage because stuck.
![image](https://github.com/KJHJason/Cultured-Downloader/assets/167768266/758117a2-3cdd-492b-b770-6dd0a5717cf0)
This is as far as i got into using the program

**Comment by KJHJason** [comment]
I'm sorry for not getting back to you sooner as I'm currently working on the GUI version of the program.
Are you able to enter any input to your command prompt?

The options are indicated in the brackets. 
For instance, (y/N) indicates that you can enter either a "y" or a "n". 
Additionally, if you press ENTER without any inputs, it will default to "n" as indicated by the capitalisation.

**Comment by KJHJason** [comment]
I just released V5.0.0 which has a Graphical User Interface and is much easier to use compared to V4!
Please check it out and let me know if you still have any issues.
